import_flow_events.py

```diff
@@ -93,69 +86,25 @@
       uid VARCHAR(64) ENCODE lzo
     );
 """
-Q_CREATE_EVENTS_TABLE = """
-    CREATE TABLE IF NOT EXISTS flow_events (
+
+Q_CREATE_EXPERIMENTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_experiments{suffix} (
+      experiment VARCHAR(40) NOT NULL DISTKEY ENCODE lzo,
+      cohort VARCHAR(40) NOT NULL ENCODE lzo,
```

Comment:
Initially i tried to call this `group` to match the nomenclature from the content server. Then I realised that clashes with the SQL keyword so decided to opt for something else. I find the need to constantly quote or namespace the `timestamp` column a bit annoying, and didn't want to add more of the same.